docs/pages/tutorial/eas/android-production-build.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Create a production build for Android
+sidebar_title: Android production build
+description: Learn about the process of creating a production build for Android.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, we're going to create our app's production version and submit it to the Google Play Store. We'll also explore how to automate the creation and release of new app versions.
+
+## Prerequisites
+
+To publish and distribute an app on the Google Play Store, we need to:
+
+- Have a paid developer account. To create one, follow the instructions on the [Google Play sign-up page](https://play.google.com/apps/publish/signup/).
+- Google Service Account email and JSON key. To create one, follow the instructions from our [expo.fyi](https://github.com/expo/fyi/blob/main/creating-google-service-account.md) guide. Once you've followed the steps, come back to this guide.
```

Comment:
This should be optional as it's only required if you want to submit automatically

Reply:
I think this our recommended path so it's might be confusing for new users as to identify what's the recommended approach here will be for them. Introducing conditions might lead to a bit more confusion.